client/header/activity-panel/index.js

```diff
@@ -254,4 +257,19 @@ class ActivityPanel extends Component {
 	}
 }
 
-export default clickOutside( ActivityPanel );
+export default withSelect( select => {
+	const { getCurrentUserData, getNotes, getNotesError, isGetNotesRequesting } = select( 'wc-api' );
+	const userData = getCurrentUserData();
+	const notesQuery = {
+		page: 1,
+		per_page: 1,
+	};
+	const latestNote = getNotes( notesQuery );
+	const unreadNotes =
+		! Boolean( getNotesError( notesQuery ) ) &&
+		! isGetNotesRequesting( notesQuery ) &&
+		latestNote[ 0 ] &&
+		new Date( latestNote[ 0 ].date_created ).getTime() > userData.activity_panel_inbox_last_read;
```

Comment:
After working on #1860, I think we will need to compare GMT dates here. Otherwise, if the store and the user's browser have a different time zone, this comparison might return unexpected results.

Reply:
Updated in inbox and the panel unread indicator to use the notes GMT time instead.